forge/routes/api/projectSnapshots.js

```diff
@@ -190,4 +190,54 @@ module.exports = async function (app) {
         const snapShot = await createSnapshot(app, project, user, snapshotProperties)
         reply.send(app.db.views.ProjectSnapshot.snapshot(snapShot))
     })
+
+    /**
+     * Export snapshot: return the full snapshot, including settings, envars, flows and re-encrypted credentials.
+     * /api/v1/projects/:instanceId/snapshots/:snapshotId/export
+     */
+    app.post('/:snapshotId/export', {
+        preHandler: app.needsPermission('project:snapshot:create'),
```

Comment:
I suggest adding a different permission scope here - `project:snapshot:export` - as exporting a snapshot outside of the platform feels like a different class of action than creating a snapshot.

You'll need to add it to `forge/lib/permissions.js` - and I suggest it should be restricted to the `Owner` role.